<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<arg name =  "pose_topic" value = "/vicon/mav_felipe/mav_felipe"/>

	<node pkg="tf" type="static_transform_publisher" name="map2world" args="0 0 0 0 0 0 1 /map /world 100" />
    <!--mav wrapper-->
    <node name="mav_wrapper_node" pkg="px4_code" type="mav_wrapper_node" output="screen">   
              
        <!--param name="mav_frame_id" value="$(arg pose_topic)"/-->
        <param name="world_frame_id" value="/world"/>
        <param name="external_pose_topic" value="$(arg pose_topic)"/>

        <param name="pose_on" value="true"/>

            
    </node>

	
    <node name="tf2pose" pkg="px4_code" type="tf2poseStamped_node" output="screen">   
	  <param name="map_frame" value="world"/>
        <param name="base_frame" value="iris__base_link"/>
	</node>
</launch>

